.gitignore

```diff
@@ -0,0 +1,2 @@
+target/
```

Comment:
Might be worth a minimal travis.yml just testing that things compile
(no need to jump the gun with things like checkstyle yet)